base/ltmiscen.dtx

```diff
@@ -276,6 +276,20 @@
 % \changes{v1.0w}{1994/11/30}{(DPC) Use \cs{@dofilelist}}
 % \changes{v1.2a}{2022/11/06}{Repeat release info at the end (gh/944)}
 %    \begin{macrocode}
+%    \end{macrocode}
+%    If we roll back we have to drop stuff before adding chunks,
+%    otherwise the code will just be appended, i.e., doubled.
```

Comment:
. . . appended, and thus doubled.

Reply:
appending is not the same as doubling!  So?

Reply:
no reason, I made some other change to clarify and overlooked that one.